packages/start/node/fetch.js

```diff
@@ -138,7 +138,9 @@ class NodeRequest extends BaseNodeRequest {
 }
 
 export function createRequest(req) {
-  let origin = req.headers.origin || `http://${req.headers.host}`;
+  let origin = req.headers.origin && 'null' !== req.headers.origin
```

Comment:
**NOTE** : I am not a maintainer or contributor to this project in any way, was just using this framework and felt like I can help by reviewing prs as I don't know internal code yet 😅 

Here we are assigning origin variable a value after checking for 'null' values ?

Why do we need to do an `&&` with req.headers.origin it can be simple comparison.

```js
'null' !=== req.headers.origin
```

Am i missing something here 😅 ?

Reply:
`req.headers.origin` can be `undefined` or `null`.